#Parameters for Admittance Controller
final_vel_topic: /RosAria/cmd_vel
virtual_wrench_topic: /virtual_wrench
human_wrench_topic: /human_wrench
shared_wrench_topic: /shared_wrench
update_params_service: /update_parameters
control_parameters:
  rate: 100
  mode: user
  mass: 8
  ldaming: 10
  inertia: 5
  adamping: 20
angular_vel_limit: 0.3
linear_vel_limit: 0.3
caraca: [-0.3, -0.1, 0.1, 0.3]
